wildfly82/README.md

```diff
@@ -0,0 +1,24 @@
+Artificer: Wildfly 8.2
+======================
+
+Artificer is an metadata and software artifact repository comprised of a common data model, multiple interfaces, powerful tools, and exensibility. It implements the [OASIS S-RAMP specification](https://www.oasis-open.org/committees/tc_home.php?wg_abbrev=s-ramp) which defines the data model, query language, and an Atom REST binding.
+
+In addition to implementing the spec, Artificer provides a suite of powerful, flexible, and extensible capabilities, described below.
+
+Artificer is 100% open source -- contributions are welcome!
+
+Visit [artificer.jboss.org](http://artificer.jboss.org) to learn more.
+
+## Usage
+
+To start Artificer
+
+    docker run -it artificer/wildfly82[:tag]
```

Comment:
It would be great if you provide information on how to extend the image, as it is done with plain wildfly, switchyard, apiman, ... so the final user will know how to add the admin user if required, and add -bmanagement so it can be managed externally, as documentation.

See [this one](https://github.com/jboss-dockerfiles/switchyard/blob/master/switchyard-wildfly/README.md), and try not to be so brief -D